src/infercnvpy/tl/__init__.py

```diff
@@ -41,24 +41,24 @@ def pca(
 ) -> Union[np.ndarray, None]:
     """Compute the PCA on the result of :func:`infercnvpy.tl.infercnv`.
 
-    Thin wrapper around :func:`scanpy.tl.pca`.
```

Comment:
The correct reference is now `scanpy.pp.pca` which is why intersphinx was failing.